src/main/java/frc/robot/Constants.java

```diff
@@ -71,15 +72,17 @@ public static final class Swerve {
   }
     
   public static final class Climb {
-    public static final double MAX_UP_VOLTAGE = 0.15; // 0.4; 
-    public static final double MAX_DOWN_VOLTAGE = -0.8; // -0.8 
-    
+    public static final double MAX_UP_VOLTAGE = 0.15; 
+    public static final double MAX_DOWN_VOLTAGE = -0.8;
+    public static final int INTAKE_CURRENT_LIMITS = 50;
+    public static final int  INTAKETOOUTTAKE_CURRENT_LIMITS  = 50;
```

Comment:
The intake constants should be in the intake constants class. Currently, these two are in the climb constants class. Also, "INTAKETOOUTTAKE_CURRENT_LIMITS" does not describe which motor we are current limiting, so the names aren't making sense.